commercetools-models/src/main/java/io/sphere/sdk/orders/messages/LineItemLikeStateTransitionMessage.java

```diff
@@ -0,0 +1,37 @@
+package io.sphere.sdk.orders.messages;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import io.sphere.sdk.messages.MessageDerivateHint;
+import io.sphere.sdk.models.Reference;
+import io.sphere.sdk.orders.Order;
+import io.sphere.sdk.states.State;
+
+import java.time.ZonedDateTime;
+
+/**
+ * This message is the result of the {@link io.sphere.sdk.orders.commands.updateactions.TransitionLineItemState} update action.
+ *
+ * {@include.example io.sphere.sdk.orders.commands.OrderUpdateCommandIntegrationTest#transitionLineItemState()}
+ *
+ */
+@JsonDeserialize(as = LineItemLikeStateTransitionMessage.class)//important to override annotation in Message class
+public final class LineItemLikeStateTransitionMessage extends LineItemLikeStateTransition {
```

Comment:
@ccsalazarr-commercetools I think you want to name it `LineItemStateTransitionMessage`, it contains a `Like` too much.